cmd/neofs-cli/modules/util/sign_session.go

```diff
@@ -38,8 +40,25 @@ func signSessionToken(cmd *cobra.Command, _ []string) {
 		common.ExitOnErr(cmd, "", errors.New("missing session token flag"))
 	}
 
-	var stok session.Object
-	common.ReadSessionToken(cmd, &stok, signFromFlag)
+	type iTokenSession interface {
+		json.Marshaler
+		json.Unmarshaler
+		Sign(ecdsa.PrivateKey) error
+	}
+	var errLast error
+	var stok iTokenSession
+
+	for _, el := range [...]iTokenSession{
+		new(session.Object),
+		new(session.Container),
+	} {
+		errLast = common.ReadSessionTokenErr(el, fPath)
+		if errLast == nil {
+			stok = el
```

Comment:
Shouldn't we `break` after?

Do we need a loop here at all? Seems like 2 simple if's will do the job.

Reply:
Added `break`. There can be more contexts, `for` allows to easily add new ones.